notebooks/samples/monitoring/conf.json

```diff
@@ -1,7 +1,7 @@
 {
     "Period": "Week",
     "TrainData": {
-        "NeomarilTrainingExecution": "1",
+        "MLOpsTrainingExecution": "1",
```

Comment:
will it also change in the backend in the future?

Reply:
Actually we dont send this anymore (I did a big find/replace)

Reply:
https://github.com/datarisk-io/mlops_codex/pull/64/commits/722fb9b1c3a4d10ced18f26d090419fb6d1cc826

Reply:
ok, is there a issue to update the samples in neomaril repo? I checked and some examples still use this variable

Reply:
I think we do.